### PR TITLE
Charts Header Refactor & 🤖 Styling

### PR DESCRIPTION
**Pull request by skylarbarrera**
Refactored Charts Header & updated styles for Android. 
Could use a comb over but this is an attempt at getting some design parity between Android & iOS 

Things fixed/tweaked: 
- Expanded State Sheets
- Restore & Backup Heights
- Unique Token bottomBorderRadius
- Gas Selection Button Width

Pixel:
Before:
http://cloud.skylarbarrera.com/Screenshot_20201218-150450.png
http://cloud.skylarbarrera.com/Screenshot_20201218-150514.png

After:
http://cloud.skylarbarrera.com/Screenshot_20201218-145531.png
http://cloud.skylarbarrera.com/Screenshot_20201218-145550.png

Samsung:
Before:
http://cloud.skylarbarrera.com/Screenshot_20201218-131024_Rainbow-1-.jpg
http://cloud.skylarbarrera.com/Screenshot_20201218-131039_Rainbow.jpg

After:
http://cloud.skylarbarrera.com/Screen_Recording_20201218-125006.mp4
http://cloud.skylarbarrera.com/Screenshot_20201218-124529_Rainbow.jpg

iOS Regression Check: http://cloud.skylarbarrera.com/Screen-Recording-2020-12-18-14-57-24.mp4

### CODE REVIEW

File: src/components/expanded-state/ChartExpandedState.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { find } from 'lodash';
 import React, { useRef } from 'react';
+import { StatusBar } from 'react-native';
+import { isSamsungGalaxy } from '../../helpers/samsung';
```
Comment (osdnk) [inline src/components/expanded-state/ChartExpandedState.js:4]: I'm a bit opposite of overusing this pattern. You know, there are thousands of devices and Samsungs are not very special in the group

File: src/components/expanded-state/ChartExpandedState.js [hunk 2]
```diff
@@ -20,11 +20,11 @@ import { ChartPathProvider } from '@rainbow-me/animated-charts';
 import AssetInputTypes from '@rainbow-me/helpers/assetInputTypes';
 
 //add's StatusBar height to android
-const heightWithoutChart = 309 + (android && 24);
-const heightWithChart = heightWithoutChart + 297;
+let heightWithoutChart = 309;
```
Comment (osdnk) [inline src/components/expanded-state/ChartExpandedState.js:23]: can we make it w/o `let`?

File: src/components/expanded-state/LiquidityPoolExpandedState.js [hunk 3]
```diff
@@ -18,11 +18,10 @@ import { ChartPathProvider } from '@rainbow-me/animated-charts';
 import { useChartThrottledPoints } from '@rainbow-me/hooks';
 import { magicMemo } from '@rainbow-me/utils';
 
-const heightWithoutChart = 373 + (android && 80);
-const heightWithChart = heightWithoutChart + 292;
+let heightWithoutChart = 373;
```
Comment (osdnk) [inline src/components/expanded-state/LiquidityPoolExpandedState.js:21]: +

File: src/components/expanded-state/chart/chart-data-labels/ChartPercentChangeLabel.js [hunk 4]
```diff
@@ -15,12 +15,12 @@ const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
 
 const PercentLabel = styled(AnimatedTextInput)`
   ${fontWithWidth(fonts.weight.bold)};
-  background-color: white;
+  background-color: transparent;
```
Comment (osdnk) [inline src/components/expanded-state/chart/chart-data-labels/ChartPercentChangeLabel.js:18]: can you import from `colors`?

**Review by osdnk — APPROVED** [review]

File: src/components/expanded-state/chart/chart-data-labels/ChartPriceLabel.js [hunk 5]
```diff
@@ -89,7 +90,7 @@ export default function ChartPriceLabel({
           return formatted;
         }}
         style={{
-          width: '100%',
+          width: android ? '67%' : '75%',
         }}
```
Comment (brunobar79) [inline src/components/expanded-state/chart/chart-data-labels/ChartPriceLabel.js:94]: same, you can pass a prop and use it to write the css dynamically in the styled component

**Review by brunobar79 — APPROVED** [review]
LGTM!  just added some minor non blocking comments

**Pull request merged**